aws_adfs/login.py

```diff
@@ -75,10 +76,14 @@
     "--username-password-command",
     help='Read username and password from the output of a shell command (expected JSON format: `{"username": "myusername", "password": "mypassword"}`)',
 )
+@click.option(
+    "--mfa-token-command",
+    help='Read MFA token from the output of a shell command (expected JSON format: `{"mfa_token": "123654"}`)',
```

Comment:
Maybe specify that is this only supported by Symantec and RSA authenticators?

Reply:
done. thanks